src/components/OrderBookWidget.stories.tsx

```diff
@@ -17,7 +17,7 @@ const defaultParams: OrderBookProps = {
   baseToken: baseTokenDefault,
   quoteToken: quoteTokenDefault,
   networkId: defaultNetworkId,
-  hops: ORDER_BOOK_HOPS_DEFAULT,
+  hops: undefined,
```

Comment:
No that there are 30 hops max, I wonder if it will be better to change `control.type`
![pr1622--dexreact review gnosisdev com_storybook__path=_story_orderbook-sample-data--liquid-market](https://user-images.githubusercontent.com/5121491/99375538-e4b24c00-28d4-11eb-9b13-4d7e7ba7b73d.png)

Reply:
Good one!